openstack/cdn/v1/domains/results.go

```diff
@@ -59,6 +59,45 @@ type CdnDomain struct {
 	ModifyTime time.Time `json:"-"`
 }
 
+type DomainSourcesDetail struct {
```

Comment:
add annotations for structure and parameters

Reply:
I prefer to keep the status quo.